libs/components/src/lib/nav-disclosure/nav-disclosure.spec.ts

```diff
@@ -142,20 +143,35 @@ describe('vwc-nav-disclosure', () => {
 				).not.toBeNull();
 			});
 		});
+	});
 
-		describe('nav-disclosure appearance', function () {
-			it('should set the appearance class on the base', async function () {
-				const appearance = 'ghost-light';
+	describe('nav-disclosure appearance', function () {
+		it('should set the appearance class on the base', async function () {
+			const appearance = 'ghost-light';
 
-				(element as any).appearance = appearance;
-				await elementUpdated(element);
+			(element as any).appearance = appearance;
+			await elementUpdated(element);
 
-				expect(
-					element?.shadowRoot
-						?.querySelector('.control')
-						?.classList.contains(`appearance-${appearance}`)
-				).toBeTruthy();
-			});
+			expect(
+				element?.shadowRoot
+					?.querySelector('.control')
+					?.classList.contains(`appearance-${appearance}`)
+			).toBeTruthy();
+		});
+	});
+
+	describe('nav-disclosure connotation', function () {
```

Comment:
By our conventions it should be the attribute name
```suggestion
	describe('connotation', function () {
```